### ISSUE TITLE
Impossible to use update method with EasyStatement

### ISSUE THREAD

**Issue by ARACOOOL** [paragonie/easydb#70]
Can you add an opportunity to use `EasyStatement` with the `update` method?
For example:
```php
$db->update(
    'database_table',
    ['event_status' => self::EVENT_STATUS_PROCESSING],
    EasyStatement::open()->in('event_id IN (?*)', $ids)
);
```

**Comment by paragonie-scott** [comment]
Wow, I feel silly. That's such an obvious use-case and I never thought of it. 👍

**Comment by paragonie-scott** [comment]
This is fixed in v2.6.0
